docroot/modules/custom/an_related_articles/templates/block--an-related-articles.html.twig

```diff
@@ -0,0 +1,17 @@
+{#
+/**
+ * @file
+ * Twig file for related article blocks.
+ */
+#}
+<div class="col-sm-3 ws-custom--block">
+  {% if articles is not empty %}
+    <div class="article-wrapper">
+      <ul>
+        {% for article in articles %}
+          <li><a href="/node/{{ article.nid }}">{{ article.title }}</a></li>
```

Comment:
I would like to see url alias being rendered here.

Reply:
Ok, I will make changes and notify you once done.